src/Polysemy/Bundle.hs

```diff
@@ -44,8 +44,8 @@ sendBundle
   -> Sem r a
 sendBundle = hoistSem $ \u -> case decomp u of
   Right (Weaving e s wv ex ins) ->
-    injWeaving $
-      Weaving (Bundle (membership @e @r') e) s (sendBundle @e @r' . wv) ex ins
+    injWeaving $ Weaving
+      (Bundle (membership @e @r') e) s (sendBundle @e @r' . wv) ex ins
```

Comment:
Don't like this linebreak. Arguments to a `Weaving` should all be in one line, or - if that runs too long - one of the following:
```haskell
Weaving e
        s
        wv
        ex
        ins
```

```haskell
Weaving
  e
  s
  wv
  ex
  ins
```

The original doesn't run past 80 chars, so I'd personally keep it on one line.